packages/vinxi-doc/doc.js

```diff
@@ -52,9 +52,11 @@ export async function doc(src) {
 		const fromFile = path.join(fromDirectory, "noop.js");
 
 		const resolveFileName = () =>
+			// @ts-ignore "Property '_resolveFilename' does not exist on type 'typeof Module'."
 			Module._resolveFilename(moduleId, {
 				id: fromFile,
 				filename: fromFile,
+				// @ts-ignore "Property '_nodeModulePaths' does not exist on type 'typeof Module'.
```

Comment:
I'm not familiar enough with what this was trying to do outside of being utilized for openapi docs that was not possible through some other public function